### ISSUE TITLE
Multiple lineages

### ISSUE THREAD

**Issue by elhenawy** [bhattlab/StrainSifter#2]
@tamburinif @bsiranosian
If there are multiple lineages of the bacterial strain of interest in the same sample, does StrainSifter only report the most abundant lineage in the metagenome or all?

**Comment by tamburinif** [comment]
@elhenawy Yes, that is correct -- only the most abundant lineage will be reported. If you are interested in metagenomic comparison that considers major and minor alleles, I'd recommend checking out [inStrain](https://instrain.readthedocs.io/en/latest/).
